src/XML/src/FMI3/fmi3_xml_unit.c

```diff
@@ -19,7 +19,7 @@
 static const char* module = "FMI3XML";
 
 fmi3_xml_unit_t* fmi3_xml_get_unit(fmi3_xml_unit_definitions_t* ud, unsigned int  index) {
-    if(index >= fmi3_xml_get_unit_definitions_number(ud)) return 0;
+    if(index >= fmi3_xml_get_unit_definitions_number(ud)) return NULL;
```

Comment:
```suggestion
    if (index >= fmi3_xml_get_unit_definitions_number(ud)) return NULL;
```

Reply:
Fixed, all instances of that in that file